mypy_django_plugin/lib/helpers.py

```diff
@@ -69,10 +69,27 @@ def lookup_fully_qualified_sym(fullname: str, all_modules: Dict[str, MypyFile])
     else:
         module, cls_name = fullname.rsplit(".", 1)
 
-    module_file = all_modules.get(module)
-    if module_file is None:
-        return None
-    sym = module_file.names.get(cls_name)
+    parent_classes: List[str] = []
+    while True:
+        module_file = all_modules.get(module)
+        if module_file:
+            break
+        if "." not in module:
+            return None
+        module, parent_cls = module.rsplit(".", 1)
+        parent_classes.insert(0, parent_cls)
+
+    scope: Union[MypyFile, TypeInfo] = module_file
+    for parent_cls in parent_classes:
```

Comment:
`.append` and then iterating in reverse works just as good here right? Perhaps a micro optimization but should at least theoretically be faster, as average for insert is `O(n)`

Ref: https://wiki.python.org/moin/TimeComplexity


```suggestion
        parent_classes.append(parent_cls)

    scope: Union[MypyFile, TypeInfo] = module_file
    for parent_cls in reversed(parent_classes):
```

Reply:
Yes, though n is so small in the vast majority of cases as to make this mostly a matter of readability.

Reply:
Yes it probably won't be noticeable.